common/testcontainers/testcontainers.go

```diff
@@ -0,0 +1,165 @@
+package testcontainers
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"scroll-tech/database"
+	"time"
+
+	"github.com/scroll-tech/go-ethereum/ethclient"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/postgres"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+type TestcontainerApps struct {
+	postgresContainer *postgres.PostgresContainer
+	l1GethContainer   *testcontainers.DockerContainer
+	l2GethContainer   *testcontainers.DockerContainer
+
+	dbClient     *sql.DB
+	DBConfig     *database.DBConfig
+	DBConfigFile string
```

Comment:
I think this is no useful

Reply:
This is to be compatible with the original logic. I will sort out the fields in this struct.

Reply:
done